content/en/tracing/advanced/connect_logs_and_traces.md

```diff
@@ -119,6 +119,26 @@ This enables automatic trace ID injection for `winston`, `bunyan`, and `pino`.
 
 **Note**: Automatic injection only works for logs formatted as JSON.
 
+{{% /tab %}}
+{{% tab ".NET" %}}
+
+Enable injection in the .NET Tracer’s [configuration][1] by setting `DD_LOGS_INJECTION=true` through environment variables or configuration files, or in code:
+
+```csharp
+using Datadog.Trace;
+
+Tracer.Instance.Settings.LogsInjectionEnabled = true;
+```
+
+The .NET Tracer uses the [LibLog][2] library to integration into your application logs. It contains transparent built-in support for [NLog][3], [Log4Net][4], [Serilog][5] and [Loupe][6].
```

Comment:
While LibLog supports Loupe, Loupe does not support the Mdc feature (Mapped Diagnostics Context) feature, so we can't instrument that logging framework. We should somehow note that Loupe will not be automatically instrumented.

Reply:
The Java page currently has this disclaimer. Perhaps we could do something similar:

**Note**: Currently only **slf4j** is supported for MDC autoinjection.

Reply:
I removed the reference to Loupe in 5e09060cda4945368f8bf6d890f5f1f04740c92e.